### ISSUE TITLE
Parent Color Wanted

### ISSUE THREAD

**Issue by srd-software** [Alexey-T/ATTabs#35]
OK, one more for today!

I really need to be able to sometimes set the tabset to its parent color, and have that change if/when the parent color changes. So I added the ParentColor back into the property list at the top of the file. Then I changed the _DoPaintBgTo_ as shown below. This seems to work well, but wasn't sure how it would fit into what you have now. Another approach might be to set up a property like ColorFromParent and handle that instead of the ParentColor property. ParentColor is easy so that's what I used. Looking for what you think about it.

```
procedure TATTabs.DoPaintBgTo(C: TCanvas; const ARect: TRect);
begin
  if ParentColor and (Parent <> nil) then
  begin
    if FColorBg <> Parent.Brush.Color then
      FColorBg := Parent.Brush.Color;
  end
  else
  begin
    if csDesigning in ComponentState then
    if FColorBg <> _InitTabColorBg then
      FColorBg := _InitTabColorBg;
  end;
  C.Brush.Color:= FColorBg;
  C.FillRect(ARect);
end;
```

Stan

**Comment by Alexey-T** [comment]
Good idea, will use

**Comment by Alexey-T** [comment]
added; plese check ParentColor.

**Comment by srd-software** [comment]
I added a pull request for some issues I found. Also, Icouldn't figure out how to delete the "Add files via upload" commit, so completely disregard that one! Just use the ParentColor Fixes commit. 

The ParentColor was only working in design mode due to ParentColor being set to false in create. I added a little code so that when the component is first added to a form, ParentColor is false. After that it uses saved settings. Also fixed spelling of "AOwner" in create, and an inline compiler error.
